erdgo/wallet.go

```diff
@@ -0,0 +1,136 @@
+package erdgo
+
+import (
+	"crypto/hmac"
+	"crypto/sha512"
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/pem"
+	"io/ioutil"
+	"os"
+
+	"github.com/ElrondNetwork/elrond-go/crypto/signing"
+	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519"
+	"github.com/tyler-smith/go-bip39"
```

Comment:
Add a README.md where you also list all these external dependencies? (which aren't ours, and which are critical)

Reply:
Done